### ISSUE TITLE
Suspicious Sniffers can't spawn bastion loot

### ISSUE THREAD

**Issue by E8zEbo8Luna** [BluePsychoRanger/SkyBlock_Collection#17]
Hello, I can't get sniffers to spawn loot in bastions, I created have made a platform in creative and put multiple sniffers on it (also checked it was in the bounding box using miniHUD). After running it for a hour (carpet mod) the sniffers did create suspicious blocks, however none of them had the bastion loot table. Some of them had the trail ruins or pyramid loot tables. This could be caused by the condition for bastion loot (~/data/skyvoid_additions/loot_tables/technical/suspicious_sniffers) not working as some of the blocks had air inside of them (/data) and by testing it via /loot sometimes a stick wouldn't spawn (a stick would still spawn other times).

TL;DR: Sniffers can create suspicious blocks in bastions, but they don't have the correct loot table, could be caused by the bastion loot condition in the datapack (~/data/skyvoid_additions/loot_tables/technical/suspicious_sniffers)

Thank you!

**Comment by E8zEbo8Luna** [comment]
I was using 1.20.1

**Comment by BluePsychoRanger** [comment]
I just tested MC 1.20.1 with Sky Void Additions v1.1.5 and bastion loot was able to spawn in the suspicious gravel. Make sure you're within the "building" bounding boxes because minihud shows 2 different bounding boxes: the full cuboid that encapsulates the structure, and the actual structure boxes that is each "room." Bastion loot will only generate "inside the rooms." For example in the image below, while the red concrete is within the full cuboid, it's not within a bastion room and thus Minecraft doesn't count it as being inside the structure.
<img width="852" alt="Screenshot 2024-04-20 at 12 59 31 PM" src="https://github.com/BluePsychoRanger/SkyBlock_Collection/assets/40612230/d5fcdd58-536b-40f4-8540-e23da6306ee8">
